wandb/wandb_controller.py

```diff
@@ -738,6 +738,43 @@ def _sweep_status(sweep_obj, sweep_conf, sweep_runs):
 
 
 def sweep(sweep, entity=None, project=None):
+    """Register a specified sweep with the W&B backend.
```

Comment:
I think this is a bit too terse/technical -- is it inaccurate to say something like "Starts up a hyperparameter sweep on the W&B backend."? I definitely want to say that 1) it's a _hyperparameter_ sweep and 2) the sweep is "started up" (or "set up"). I also don't know that we need to mention the backend.

I would then add a slightly longer description beneath, before diving into the parameters, something like:

> Starts up a sweep controller that produces new hyperparameter combinations to try, based on the provided configuration or configuration generator `sweep`. To start using those hyperparameters in experiments, call `wandb.agent` with the `sweep_ID` returned by this command. Intended for use in Jupyter environments. For executing sweeps with Python scripts, see the command line tool [wandb sweep](https://docs.wandb.ai/ref/cli/wandb-sweep).

Reply:
Updated. I went with "Initialize a hyperparameter sweep." for the toplevel docstring. 

> Intended for use in Jupyter environments. For executing sweeps with Python scripts, see the command line tool wandb sweep.

I punted on including this because I could see this function being very convenient to use a python script (e.g., the example script in the docstring).